### PR TITLE
modified Map, Navbar and StationMiniCard components

### PR DESCRIPTION
**Pull request by JulienOza**
modified Map, Navbar and StationMiniCard components:
- display stations rather than charging points
- calculating distance to stations from user's position, display nearest stations sorted by distance

### CODE REVIEW

**Review by SebSamPro — APPROVED** [review]

File: frontend/src/components/CarShow/AddCarModal/AddCarModal.scss [hunk 1]
```diff
@@ -123,17 +123,17 @@ fieldset {
...
-    opacity: 1;
+    opacity: 60%;
   }
 }
 
 @keyframes contentShow {
   from {
     opacity: 0;
-    transform: translate(-50%, -48%) scale(0.96);
+    // transform: translate(-50%, -48%) scale(0.96);
```
Comment (ItsMatDev) [inline frontend/src/components/CarShow/AddCarModal/AddCarModal.scss]: remove comment

File: frontend/src/components/CarShow/AddCarModal/AddCarModal.scss [hunk 2]
```diff
@@ -123,17 +123,17 @@ fieldset {
...
   }
   to {
     opacity: 1;
-    transform: translate(-50%, -50%) scale(1);
+    // transform: translate(-50%, -50%) scale(1);
```
Comment (ItsMatDev) [inline frontend/src/components/CarShow/AddCarModal/AddCarModal.scss]: here too

**Review by ItsMatDev — CHANGES_REQUESTED** [review]

**Review by Bravax — APPROVED** [review]

File: frontend/src/components/CarShow/AddCarModal/AddCarModal.scss (hunk 1 above)
Comment (JulienOza) [reply frontend/src/components/CarShow/AddCarModal/AddCarModal.scss]: done

File: frontend/src/components/CarShow/AddCarModal/AddCarModal.scss (hunk 2 above)
Comment (JulienOza) [reply frontend/src/components/CarShow/AddCarModal/AddCarModal.scss]: done

**Review by Yoann-Fortin — APPROVED** [review]

**Review by ItsMatDev — APPROVED** [review]

**Pull request merged**